app/views/layouts/orgchart.html.erb

```diff
@@ -1,173 +1,128 @@
+      }
+      li.lieutenantX {
+        width: 130pt;
+        margin: 5pt 0 0 62pt;
+      }
+      ul#division1 {
+       left: 7pt;
+      }
 
-ul#division2 {
- right: 7pt;
-}
+      ul#division2 {
+       right: 7pt;
+      }
 
-.squads ul {
-  padding: 0;
-  margin: 16pt 0;
-  top: 75pt;
-}
+      .squads ul {
+        padding: 0;
+        margin: 16pt 0;
+        top: 75pt;
+      }
 
-ul.squad1 {
-  left: 7pt;
-}
-ul.squad2 {
-  right: 7pt;
-}
-li.squads {
-  border: 0;
-}
-.division_heading,
-.squad_heading {
-  font-weight: bold;
-  font-size: 115%;
-}
+      ul.squad1 {
+        left: 7pt;
+      }
+      ul.squad2 {
+        right: 7pt;
+      }
+      li.squads {
+        border: 0;
+      }
+      .division_heading,
+      .squad_heading {
+        font-weight: bold;
+        font-size: 115%;
+      }
 
-.chief,
-.deputy,
-.captain,
-.lieutenant,
-.sargeant,
-.corporal,
-.senior,
-.division_heading,
-.squad_heading {
-  border: 4px solid #555;
-}
-
-</style>
-<h1></h1>
-<h2>Department Roster (<%= Date::today().to_s %>)</h2>
-<ul class="senior_staff">
-  <% Person.active.divisionC.each do |p| %>
-       <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-  <% end %>
-</ul>
-
-
-<div class="divisions">
-<ul class="division" id="division1" >
-  <li class="division_heading">Division 1</li>
-  <% Person.active.division1.squadC.each do |p| %>
-    <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-  <% end %>
-  <li class="squads">
-    <ul class="squad1">
-      <li class="squad_heading">Squad 1</li>
-      <% Person.active.division1.squad1.each do |p| %>
-        <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-      <% end %>
-    </ul>
-
-    <ul class="squad2">
-      <li class="squad_heading">Squad 2</li>
-      <% Person.active.division1.squad2.each do |p| %>
-        <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-      <% end %>
-    </ul>
-  </li>
-</ul>
-
-<ul class="division" id="division2" >
-  <li class="division_heading">Division 2</li>
-  <% Person.active.division2.squadC.each do |p| %>
-    <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-  <% end %>
-  <li class="squads">
-    <ul class="squad1">
-      <li class="squad_heading">Squad 1</li>
-      <% Person.active.division2.squad1.each do |p| %>
-        <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-      <% end %>
-    </ul>
-
-    <ul class="squad2">
-      <li class="squad_heading">Squad 2</li>
-      <% Person.active.division2.squad2.each do |p| %>
-        <%= render :partial => 'coc_boxes', :locals => { :p => p} %>
-      <% end %>
-    </ul>
-  </li>
-</ul>
-<span id="thedate" > <%#= Date::today().to_s %></span>
-</div>
+      .chief,
+      .deputy,
+      .captain,
+      .lieutenant,
+      .sargeant,
+      .corporal,
+      .senior,
+      .division_heading,
+      .squad_heading {
+        border: 4px solid #555;
+      }
+    </style>
+  </head>
+  <body>
+    <%= yield %>

```

Comment:
previously, this layout ignored the actual view template being rendered. I've added a `yield` here to address this.